dask/kubernetes/specs/sched-spec.yaml

```diff
@@ -1,19 +1,25 @@
+apiVersion: v1
 kind: Pod
 metadata:
+  name: dask-scheduler
   labels:
+    cluster_type: dask
     dask_type: scheduler
 spec:
   restartPolicy: Never
   containers:
-    - image: gcr.io/[YOUR_GCP_PROJECT]/[YOUR_USER_ID]/dask-unified:21.06
-      imagePullPolicy: IfNotPresent
+  # TODO: update image to official image
+    - image: isvoid/rapidsai-cloud-worker:22.06-cuda11.5-runtime-ubuntu20.04-py3.9
```

Comment:
Where does this Docker image get built?

Reply:
https://hub.docker.com/r/isvoid/rapidsai-cloud-worker
Just a simple image adding `gcsfs` in the environment.

Reply:
Sure, but I mean it is used in the examples but never mentioned where it came from.

Reply:
If this custom image is installing additional packages you should be able to use the base RAPIDS docker images and set the `EXTRA_CONDA_PACKAGES` or `EXTRA_PIP_PACKAGES` environment variables to install these packages at runtime.

Reply:
Hmm, adding below to the specs doesn't seem to work:
```
      - name: EXTRA_PIP_PACKAGE
        value: "gcsfs"
```
I tried both PIP/CONDA on both scheduler and worker.

Reply:
There's a typo in the above command. It should be `EXTRA_PIP_PACKAGES`.